ios/browser/api/debounce/debounce_service.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_IOS_BROWSER_API_DEBOUNCE_DEBOUNCE_SERVICE_H_
+#define BRAVE_IOS_BROWSER_API_DEBOUNCE_DEBOUNCE_SERVICE_H_
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+OBJC_EXPORT
+@interface DebounceService : NSObject
+- (instancetype)init NS_UNAVAILABLE;
+
+/**
+ True or false if debounce is enabled
+ */
+- (bool)isDebounceEnabled;
```

Comment:
This can just be `isEnabled` or `@property(readonly) bool isEnabled;` no need for debounce part

Reply:
Changed

Reply:
Actually this was a mistake on my end. I should be exposing the setter too and getting/setting this new value in iOS. though this doesn't really need to be done right away because the default is set to true and the setting is unused within the actual service. But in case this changes, I will include the setter in this PR. Please take a look when you get a chance.

I apologize for the confusion.